utils/litemap/src/testing.rs

```diff
@@ -111,21 +122,69 @@ where
 /// Call this function in a test and pass it an empty instance of a `LiteMap` with a custom store.
 // Test code
 #[allow(clippy::expect_used)]
-pub fn check_litemap<'a, S>(mut litemap_test: LiteMap<u32, u64, S>)
+pub fn check_litemap<'a, S>()
```

Comment:
Thought: we're testing the `Store`s here, so we should call this `check_store`

Reply:
Yes renaming `check_litemap` to `check_store` makes sense.